### DISCUSSION TITLE
How do create an email account password？

### DISCUSSION THREAD

**Discussion by slrem** [emersion/maddy#402] [Q&A]
How do create an email account password？

**Comment by foxcpp** [comment]
```
maddyctl creds create <account_name>
```
What exactly is your issue?

**Reply by slrem** [reply]
![image](https://user-images.githubusercontent.com/31646684/133886109-d3988f27-2df5-482c-974a-4ac74133fd76.png)
Copy to host or container?

**Reply by foxcpp** [reply]
/data/tls_* should be in the container.

**Reply by slrem** [reply]
but the docker container start failed ，How do I copy into a container？

**Reply by foxcpp** [reply]
You can copy into volume's FS directly, it is in /var/lib/docker/volumes/maddydata.

**Reply by slrem** [reply]
I go to try！

**Reply by slrem** [reply]
the container is runing and The account has been added.   but Can't connect，can you help me ?
foxmail UI:
![image](https://user-images.githubusercontent.com/31646684/134892447-47085acc-3dc4-4de8-b849-d06dc2ae48f9.png)
DNS configuration:
![image](https://user-images.githubusercontent.com/31646684/134892545-b10d7729-59e1-424f-b4ec-a0b11d58e28e.png)

**Reply by foxcpp** [reply]
SMTP port should be 587, maddy does not support submission over 25

**Reply by slrem** [reply]
It still doesn't work.
![image](https://user-images.githubusercontent.com/31646684/134895954-191eb023-024d-4e30-aed8-be6ad5add132.png)

**Reply by foxcpp** [reply]
SSL probably should be checked.

**Reply by slrem** [reply]
How to Configure SSL？
![image](https://user-images.githubusercontent.com/31646684/134897265-26120168-7049-4bf5-87e1-356181f03264.png)

**Reply by xdkaka** [reply]
> SSL probably should be checked.

My server is only used for sending mail. I have many servers that don't want to configure SSL for each domain name. I can't find the code to verify SSL to cancel verify. How can I change the code to implement port 25 without SSL verification?

like this
````
submission tls://0.0.0.0:465 tcp://0.0.0.0:587 tcp://0.0.0.0:25 {
    debug yes
    io_debug yes
    
    auth &local_authdb

    .....
}

````
